wiki/Reporting_Bad_Behaviour/Handling_Cheating/en.md

```diff
@@ -1,74 +1,72 @@
 
-I have been banned for cheating. I didn't cheat!
-------------------------------------------------
 
+## I have been banned for cheating. I didn't cheat!
 osu! uses an automated metrics combined with manual confirmation.
 
 The staff never places an incorrect ban for cheating. Before they do any actions, they make sure it first 100%.
 
-My friend has been banned for cheating! They definitely wouldn't cheat! You are wrong!
---------------------------------------------------------------------------------------
 
+## My friend has been banned for cheating! They definitely wouldn't cheat! You are wrong!
 It may be hard to come to terms with, but you will have to trust the staff. peppy is not trying to tear apart friendships but he knows that nearly no one can be trusted. Even those people who are close to you that you may never suspect can be just acting behind you or any other person.
 
-Prove that I/they were cheating! Tell us how you know!
-------------------------------------------------------
 
+## Prove that I/they were cheating! Tell us how you know!
 The staff cannot say on how can they prove. Giving away the countermeasures would make them useless in further detecting cheaters.
 
-I'll make a video to prove my innocence!
-----------------------------------------
 
+## I'll make a video to prove my innocence!
 Sorry but if you have been banned, they are already 100% sure you are cheating. A video will not help.
 
-I found someone who is definitely cheating. I can tell by their replay/score! What should I do?
------------------------------------------------------------------------------------------------
 
-Stay calm.
+## I found someone who is definitely cheating. I can tell by their replay/score! What should I do?
+Stay calm and **DON'T discuss their play in public chat**.
+Don't give them undue attention or ask others to spectate them. There are three ways to report users.
 
-**DON'T discuss their play in public chat**.
+### User Profile
+At the user's profile, a "Report User" button will appear when you move your cursor to their avatar. Write any information that can prove that the user is doing misbehaviour.
 
-Don't give them undue attention or ask others to spectate them.
+*Take note of the selected game mode before playing*
 
-Click the "Report" link at the right side of their score on the beatmap info (website)
+### Beatmap Page
+In beatmap pages, you can report a user in the top leaderboards by clicking the "Report" button at right side.
 
-Use !report in in-game chat to send a message to moderators.
+### In-game chat
+Use the `!report` command in any channel to inform the chat moderators. Other players will not see your reports in chat.
 
-`Example: !report `<Username>` is suspected to be a cheater.`
+      Format: !report <username> <reason>
+     Example: !report this_user suspected cheating
 
 Your help in spotting foul play is much appreciated. You can also send an email directly to osu@ppy.sh for a relatively fast response.
```

Comment:
add mailto link
( not sure what it is though)

Reply:
Most browsers automatically detect email addresses and add the mailto links themselves.

Reply:
really? I don't think so 😕